contrib/kube-prometheus/manifests/prometheus/prometheus-k8s-rules.yaml

```diff
@@ -516,4 +516,52 @@ data:
         summary = "Kubelet is close to pod limit",
         description = "Kubelet {{$labels.instance}} is running {{$value}} pods, close to the limit of 110",
       }
-    
+
+    ALERT K8SPodWaiting
+      IF sum(sum_over_time(kube_pod_container_status_waiting[5m]) > 0 ) BY (job, exported_pod, instance, namespace)
```

Comment:
Why do we sum over time here before summing over containers? This would trigger even if *right now* there's no waiting container at all.